src/status_im/utils/gfycat/core.cljs

```diff
@@ -1,6 +1,7 @@
 (ns status-im.utils.gfycat.core
   (:require [status-im.utils.gfycat.animals :as animals]
             [status-im.utils.gfycat.adjectives :as adjectives]
```

Comment:
dictionaries aren't used anymore? can we delete them?

Reply:
yes, I was waiting to remove libraries and dictionaries until we made a decision about desktop